src/analysisd/logtest.c

```diff
@@ -42,7 +47,7 @@ void *w_logtest_init() {
         w_create_thread(w_logtest_main, &connection);
     }
 
-    w_logtest_main(&connection);
+    w_create_thread(w_logtest_check_active_sessions, NULL);
```

Comment:
if delete this line:
`w_logtest_main(&connection);`
Then one less main_init thread will be created
if w_logtest_conf.threads==1, there will be no threads to attend connections.
I suggest change this
`for (int i = 1; i < w_logtest_conf.threads; i++) `
for this
`for (int i = 1; i  =< w_logtest_conf.threads; i++)`

Reply:
I regret,i suggest add this line after lauch w_logtest_check_active_sessions thread
`w_logtest_main (&connection);`
this cause the socket not to close and remove

Reply:
Exactly. After testing to make sure everything works properly, I removed that line by accident.
Thank you very much @juliancnn  !